app/controllers/api/base_controller/action.rb

```diff
@@ -77,23 +88,15 @@ def queue_object_action(object, summary, options)
           :role        => options[:role] || nil,
         }
 
-        queue_options.merge!(options[:user]) if options.key?(:user)
+        if options.key?(:user)
```

Comment:
Since we require `:user => true` via the interface and technically that value can be overridden...we should check the actual value.  (i.e. `{:user => false}.key?(:user) == true`

```suggestion
        if options[:user]
```

Reply:
done